lib/config/presets/internal/schedule.ts

```diff
@@ -2,9 +2,9 @@ import type { Preset } from '../types';
 
 /* eslint sort-keys: ["error", "asc", {caseSensitive: false, natural: true}] */
 
-const daily = ['before 2am'];
-const earlyMondays = ['before 3am on Monday'];
-const monthly = ['before 3am on the first day of the month'];
+const daily = ['before 4am'];
+const earlyMondays = ['before 4am on Monday'];
+const monthly = ['before 4am on the first day of the month'];
```

Comment:
Copy/pasting this from the PR body, so I know it's seen by you maintainers. 😉 

> We have presets that rely on (extend) the `schedule:earlyMondays` or `schedule:monthly` presets. Do you want me to label this a breaking change, and target our `v36` branch, so we merge it as a breaking change?

Reply:
I think this is non-breaking